### PR TITLE
routing and components structured; AuthFactory set

### PR DESCRIPTION
**Pull request by iiimosley**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -1,9 +1,63 @@
...
+        let creds = FBCreds;
+        let authConfig = {
+            apiKey: creds.apiKey,
+            authDomain: creds.authDomain
+        };
+        firebase.initializeApp(authConfig);
+    });
 
 
 require('./factories');
```
Comment (TimAConner) [inline js/app.js:63]: Add require('./values/fbcreds'); below these to pull in the constants!

Comment (iiimosley) [reply js/app.js:63]: 👍 👀

**Comment by kates-code** [comment]
💯

**Comment by melissabell456** [comment]
👍

**Pull request merged**